### PR TITLE
Add CI checks for migrations and translations

### LINKED ISSUE #184: Refresh translations + add CI check
I recreated translations last week and noticed many changes.

Files should be regenerated so that translators can update translations.
I will contribute French!

### PR DESCRIPTION
**Pull request by merwok**
fixes #184

### CODE REVIEW

**Comment by merwok** [comment]
A tox config was already present (if unused?), so I added test commands there rather than introducing pre-commit.
Up to you to adapt or transform this!

File: update-translations.sh [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+for lang in es fa fr it pl pt_BR tr; {
+    python -m django makemessages -l $lang;
```
Comment (merwok) [inline update-translations.sh:3]: I tried adding all languages to test settings to make `makemessages --all` work, but got an error about the code `pt_BR`

**Comment by fabiocaccamo** [comment]
> A tox config was already present (if unused?), so I added test commands there rather than introducing pre-commit. Up to you to adapt or transform this!

The `tox` config is old, I used it before with TravisCI, then I replaced both with GH workflow.

File: update-translations.sh (hunk 1 above)
Comment (fabiocaccamo) [reply update-translations.sh:3]: Really strange ...

**Review by fabiocaccamo — APPROVED** [review]

**Pull request merged**